docs/assets/fragments/connectivity.txt

```diff
@@ -0,0 +1,45 @@
+1. You will need the login and password for the admin user to access the
+    cluster. Use `kubectl get secrets` command to see the list of Secrets
+    objects (by default the Secrets object you are interested in has
+    `my-cluster-name-secrets` name). Then
+    `kubectl get secret my-cluster-name-secrets -o yaml` command will return
```

Comment:
I'm not sure that we need to have this file as part of the documentation

Reply:
Removed it.